jooby/src/main/java/io/jooby/internal/HashValue.java

```diff
@@ -26,11 +26,11 @@
 import java.util.function.BiConsumer;
 
 public class HashValue implements ValueNode, Formdata {
```

Comment:
What if we allow hashvalue to get a `Supplier<Map<String, ValueNode>>` at construction level, then we add a `Value.headers` static method and from there we provide a `TreeHashMap`?

Reply:
changed as requested